File: NetFlexAPI/NetFlexAPI/Models/Review.cs
namespace NetFlexAPI.Models;

public class Review
{
    public Guid Id { get; set; }
    public string UserName { get; set; }
    public Guid ContentId { get; set; }
    public string Text { get; set; }
    public float Rating { get; set; }
    public DateTime PublishTime { get; set; }
}

File: NetFlexAPI/GiraffeAPI/Models/Review.fs
module GiraffeAPI.Models.Review

open System
open System.ComponentModel.DataAnnotations
open Microsoft.EntityFrameworkCore.Metadata.Internal
open Ply.TplPrimitives

[<CLIMutable>]  
type Review =
    {
        Id : Guid
        UserName : string
        ContentId : Guid
        Text : string
        Rating : float
        PublishTime : DateTime
    }